shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -352,16 +343,20 @@ static bool DidFlagChange(const flutter::SemanticsNode& oldNode,
   return nil;
 }
 
-void AccessibilityBridge::HandleEvent(NSDictionary<NSString*, id>* annotatedEvent) {
-  NSString* type = annotatedEvent[@"type"];
+void AccessibilityBridge::HandleMessage(NSDictionary<NSString*, id>* message, FlutterReply reply) {
+  NSString* type = message[@"type"];
   if ([type isEqualToString:@"announce"]) {
-    NSString* message = annotatedEvent[@"data"][@"message"];
-    ios_delegate_->PostAccessibilityNotification(UIAccessibilityAnnouncementNotification, message);
+    NSString* messageToAnnounce = message[@"data"][@"message"];
```

Comment:
This should be `message_to_announce` since we're in a C++ method. I'm not in love with it either.

https://google.github.io/styleguide/objcguide.html#objective-c